### PR TITLE
[Tests-Only]Adjust proper tags for the tests in expected failures file

### PR DESCRIPTION
**Pull request by jasson99**
## Description
This Pr adjusts proper tags for the tests in expected failures file which were previously tagged with irrelevant `group share issue` tag

## Related Issue
- https://github.com/owncloud/ocis/issues/1289

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingAutocompletion</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14505/

<p>







</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>Move</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14505/

<p>










</p></details>

**Review by phil-davis — COMMENTED** [review]
why is there an empty ` config/config.json-ocis` added?

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingInternalGroupsRootSharingIndicator</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14623/

<p>




</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>SharingAutocompletion</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14631/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles2</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/14926/

<p>







</p></details>

File: tests/acceptance/helpers/sharingHelper.js [hunk 1]
```diff
@@ -250,6 +251,7 @@ module.exports = {
     params.set('format', 'json')
     params.set('state', 'all')
     const apiURL = `apps/files_sharing/api/v1/shares?${params.toString()}`
+    console.log(apiURL)
```
Comment (kiranparajuli589) [inline tests/acceptance/helpers/sharingHelper.js:254]: remove if debug line

File: tests/acceptance/helpers/sharingHelper.js [hunk 2]
```diff
@@ -115,6 +115,7 @@ module.exports = {
           'Could not get shares. Message: ' + sharesResult.ocs.meta.message
         )
         const shares = sharesResult.ocs.data
+        console.log(shares)
```
Comment (kiranparajuli589) [inline tests/acceptance/helpers/sharingHelper.js:118]: remove if debug line

File: tests/acceptance/helpers/sharingHelper.js [hunk 3]
```diff
@@ -325,7 +327,17 @@ module.exports = {
...
+     in the path and not the containing folder
+     eg: if "simple-folder/subfolder" has been shared, the share details is shown as
+     path: "/subfolder"
+     */
+    if (client.globals.ocis) {
+      const splitted = filename.split('/')
+      filename = splitted[splitted.length - 1]
+    }
     const allShares = await this.getAllSharesSharedWithUser(user)
+    console.log(allShares)
```
Comment (Talank) [inline tests/acceptance/helpers/sharingHelper.js:340]: remove the console log

File: tests/acceptance/helpers/sharingHelper.js (hunk 1 above)
Comment (jasson99) [reply tests/acceptance/helpers/sharingHelper.js]: fixed

File: tests/acceptance/helpers/sharingHelper.js (hunk 2 above)
Comment (jasson99) [reply tests/acceptance/helpers/sharingHelper.js]: fixed

File: tests/acceptance/helpers/sharingHelper.js (hunk 3 above)
Comment (jasson99) [reply tests/acceptance/helpers/sharingHelper.js]: fixed

**Review by swoichha — COMMENTED** [review]
LGTM :+1:

**Review by saw-jan — COMMENTED** [review]
LGTM

**Review by Talank — COMMENTED** [review]
LGTM

File: tests/acceptance/helpers/sharingHelper.js [hunk 4]
```diff
@@ -325,6 +325,15 @@ module.exports = {
    * @param {string} sharer
    */
   acceptShare: async function(filename, user, sharer) {
+    /** In ocis, when a resource inside a folder is shared, the share details contains only the resource
+     in the path and not the containing folder
+     eg: if "simple-folder/subfolder" has been shared, the share details is shown as
+     path: "/subfolder"
+     */
```
Comment (individual-it) [inline tests/acceptance/helpers/sharingHelper.js:332]: is this a known bug? if yes let's mention the issue here if not please open an issue

**Review by individual-it — CHANGES_REQUESTED** [review]

File: tests/acceptance/helpers/sharingHelper.js (hunk 4 above)
Comment (jasson99) [reply tests/acceptance/helpers/sharingHelper.js]: yes it is. I mentioned the issue there now.

**Review by individual-it — APPROVED** [review]

**Pull request merged**